foundation/foundation-archetype/src/main/resources/archetype-resources/devops/jenkinsPipelineSteps.groovy

```diff
@@ -41,14 +41,14 @@ def argocdSync(argocdAppName, argocdBranch) {
 
 def argocdDeploy(argocdAppName, argocdUrl, argocdDestinationServer, gitRepo, argocdBranch, argocdNamespace, values) {
     String valuesParam = "--values " + values.join(" --values ")
-    sh "sleep 60" //wait for app to shutdown before creation
     sh "/usr/local/bin/argocd app create ${argocdAppName} --grpc-web \
             --server ${argocdUrl} \
             --dest-namespace ${argocdNamespace} \
             --dest-server ${argocdDestinationServer} \
             --repo ${gitRepo} \
             --path ${artifactId}-deploy/src/main/resources --revision ${argocdBranch} \
-            ${valuesParam}"
+            ${valuesParam} \
+            --helm-set spec.targetRevision=${argocdBranch}"
```

Comment:
Q: are we use the command to override the targetRevison in the values-ci file?

Reply:
Correct. The default is still `dev`. However, should you run the argocd deploy on a different branch, you will no longer need to update the targetRevision in that file as well.